phpunit.xml.dist

```diff
@@ -33,20 +25,16 @@
             <exclude>
                 <directory>./tests</directory>
                 <directory>./vendor</directory>
-                <directory>./Resources</directory>
-                <directory>./DependencyInjection</directory>
-                <file>./SonataDoctrinePHPCRAdminBundle.php</file>
+                <directory>./src/Resources</directory>
+                <directory>./src/DependencyInjection</directory>
+                <file>./src/SonataDoctrinePHPCRAdminBundle.php</file>
                 <directory>./coverage/</directory>
             </exclude>
         </whitelist>
     </filter>
-
-    <listeners>
-        <listener class="Symfony\Cmf\Component\Testing\Phpunit\DatabaseTestListener" />
-    </listeners>
-
     <php>
         <ini name="precision" value="8"/>
+        <env name="KERNEL_CLASS" value="Sonata\DoctrinePHPCRAdminBundle\Tests\Fixtures\App\Kernel" />
```

Comment:
With `2.1.2` release KERNEL_DIR was required, but https://github.com/symfony-cmf/testing/pull/183 made possible to set `KERNEL_CLASS` only )